### ISSUE TITLE
UVR GUI crashes with error msg after 20% of job done

### ISSUE THREAD

**Issue by DDBE12** [Anjok07/ultimatevocalremovergui#132]
**My OS**
First off, I'm on Windows 10 here.

**My UVR settings**
I ran the GUI, giving it an assignment for the very first time. Here's my settings:

![settings](https://user-images.githubusercontent.com/96987155/147880543-08b5ca86-095a-4f61-8bea-33a7fa7812c1.jpg)

**How it went before the error**
When I hit "Start Conversion" on the GUI, the Anaconda prompt did every single model, running from 0-100% per model, and did that with 4 models. It was as fast as only taking about 7 minutes per model to go from 0-100%, with the WAV file being almost 4 minutes long.

What confused me was that although I had the option of "GPU conversion" checked so it's being FORCED to use my GPU (and in that case, it would give me an error msg if it couldn't find a compatible GPU, right?) instead of my CPU for as much of the task as possible, while it was busy my CPU went up to fluctuate between 60-100%, whereas my GPU didn't go above 1%. So, I killed Firefox in case it needed more RAM to work more efficiently or something, and the only result was that my GPU usage went up from 1 to 2% while it was still busy.

It went as far as this:

![Anaconda](https://user-images.githubusercontent.com/96987155/147880834-8c0f03fe-a43c-4c67-bd87-8a0d46b95732.jpg)

On the GUI, that seems to translate to about 20% of the entire job in the green bar at the bottom.

**The Error**
Immediately after finishing the 4th model, the GUI popped up an error window, and the Anaconda prompt went unresponsive. Here's the error log:

![error](https://user-images.githubusercontent.com/96987155/147880850-b07ecb3f-fc50-4c4b-b39a-03e9f43ef29c.jpg)

What may also be of importance here is that I can only run the app by running Anaconda as an admin and then run VocalRemover.py from there via cmd line.

**The Result**
Although it crashed at roughly 20% of the entire job, it had output one single file: [FILE NAME]_MGM_MAIN_v4_sr44100_hl512_nf2048-StackedMGM_MM_v4_sr44100_hl512_nf2048.wav The file is an utterly PERFECT vocals-only file!

However, I wanna avoid the error and I also want the instruments-only file.

**Comment by DDBE12** [comment]
Okay, I've fixed it now:

I'd actually *FORGOTTEN* to install Python at the start! Before I found that out, everything was running exclusively on Anaconda, which seems to have some very basic Python functionality even just on its own.

So, I went and installed Python, then installed everything else on top of it again.

First difference: VocalRemover.py still doesn't work on double-click, but now it works when run from Anaconda *EVEN* when Anaconda is not run as an admin.

However, when I'm not running Anaconda as an admin, it now crashes even sooner after but two 0-100% progress bars in Anaconda.

What finally did the trick then was to run Anaconda as an admin again, then start VR from that, and then everything works from start to finish. Now, my CPU only goes up to 60% when having UVR process stuff, not to 100% anymore, and at times, my AMD GPU can go as high as 9% during that.

Just one thing is strange: I did 7 stacks, and in the Stacked Output folder, *ALL* the isolated vocal outputs are entirely silent. However, all the instrumental outputs inside that folder are fine. Outside that folder, the final versions where all 7 stacks are kinda all "stacked on top of each other"(?) are just fine, both the instrumental and the vocal version.
